src/main/java/org/broadinstitute/hellbender/conf/GATKConfBuilder.java

```diff
@@ -0,0 +1,92 @@
+package org.broadinstitute.hellbender.conf;
+
+import org.apache.commons.configuration.AbstractConfiguration;
+import org.apache.commons.configuration.BaseConfiguration;
+import org.apache.commons.configuration.CombinedConfiguration;
+import org.apache.commons.configuration.Configuration;
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.DefaultConfigurationBuilder;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public class GATKConfBuilder {
+
+    public static final List<String> DEFAULT_TOOLS_PACKAGES =
+            Collections.singletonList("org.broadinstitute.hellbender");
+
+    public static final List<String> DEFAULT_CODEC_PACKAGES = Arrays.asList("htsjdk.variant",
+            "htsjdk.tribble",
+            "org.broadinstitute.hellbender.utils.codecs");
+
+    private AbstractConfiguration configuration;
+
+    private boolean loadFile = false;
+
+    public GATKConfBuilder() {
+        this.configuration = new BaseConfiguration();
+    }
+
+    public GATKConfBuilder(final File file) {
+        this.configuration = new DefaultConfigurationBuilder(file);
+    }
+
+    public static final GATKConf getDefaultConfiguration() {
+        return new GATKConfBuilder().setDefaults().getConfiguration();
+    }
+
+    public static final GATKConf useConfiguration(final Configuration configuration) {
+        return new GATKConf(configuration);
+    }
+
+    public GATKConfBuilder setDefaults() {
+        return setToolsPackages(new ArrayList<>(DEFAULT_TOOLS_PACKAGES))
+                .setClpClasses(new ArrayList<>())
+                .setCodecPackages(new ArrayList<>(DEFAULT_CODEC_PACKAGES));
```

Comment:
A properties file will be great, but I decided to hardcode these properties because how is it going to be handled this file in project dependencies? I'm not sure how to do this in a proper way, because I'm not coming from the IT background...I may need some help on this.